app/service.js

```diff
@@ -42,13 +42,17 @@ function getDocumentMetadata(url, window) {
     title: metadata.title,
     description: metadata.description,
     favicon_url: metadata.icon_url ? makeUrlAbsolute(url, metadata.icon_url) : makeUrlAbsolute(url, '/favicon.ico'),
-    images: [{
+    images: []
+  };
+
+  if (metadata.image_url) {
+    responseData.images = [{
```

Comment:
For example, it'd be nice to be able to have our parser support this stuff, but probably outside the scope of life if we're only building this for specific Activity Stream use cases:

```js
<meta property="og:image" content="http://example.com/rock.jpg" />
<meta property="og:image" content="http://example.com/rock2.jpg" />
```
via http://ogp.me/#array